src/Sylius/Bundle/ShopBundle/Resources/translations/messages.en.yml

```
@@ -30,10 +30,10 @@ sylius:
             verify_your_email_address: 'Verify your email address'
     homepage:
         about_us: 'About Us'
-        about_us_content: 'Lorem ipsum dolor sit amet, consectetur adipisicing elit. Ab ad adipisci aperiam consectetur cum dicta dolores ducimus, eveniet ex exercitationem harum in iure labore libero magnam minima neque, numquam officiis possimus quasi reiciendis rem repellat repellendus rerum ut vel veniam vitae? Beatae consectetur culpa debitis ex laudantium porro unde? Modi.'
-        brace_yoursel_winter_is_coming: 'Brace yourself! Winter is coming'
+        about_us_content: 'Welcome to our store, your number one online shop that gives you the very best products. You can choose from the very best clothes and shoes from our offer. (You can change this by overwriting sylius.homepage.about_us_content key in your translation file)'
```

```suggestion
        about_us_content: 'Welcome to our store, your number one online shop that gives you the very best products. You can choose from the very best clothes and shoes from our offer. (You can modify this content by overriding "sylius.homepage.about_us_content" key in your translation file)'
```

an if we could somehow use _italics_ on the sentence written in `(...)` it would be perfect. :D